protobuf/meshnet/peer.proto

```diff
@@ -167,21 +179,17 @@ enum DisableAutomaticFileshareErrorCode {
 message AllowRoutingResponse {
 	oneof response {
 		Empty empty = 1;
-		UpdatePeerErrorCode update_peer_error_code = 2;
 		AllowRoutingErrorCode allow_routing_error_code = 3;
-		ServiceErrorCode service_error_code = 4;
-		MeshnetErrorCode meshnet_error_code = 5;
+		UpdatePeerError update_peer_error = 6;
```

Comment:
Is the gap with field numbers intentional here?

Reply:
Yes, this is for client and server schema mismatches considering no extra measures are done. E. g. client has old version of schema but server has a new one, will try to find type with id `6`, will not be able to do it will produce an appropriate error. Otherwise, if this was `4`, it would try to parse the given data to `ServiceErrorCode` and would produce error that it was unable to do so.